NutriLife/src/components/SearchBar.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react"
+import { MantineProvider } from "@mantine/core";
+const searchBar = () =>{
```

Comment:
All component names should start with a capital letter. So this should be SearchBar